cmd/ontap-cvo.go

```diff
@@ -215,9 +188,10 @@ func checkSecrets(client *kubernetes.Clientset, profileName string, profileEmail
 			onPremName, foundOnPrem := getOnPrem(profileEmail, client)
 			if foundOnPrem {
 				// Create the user
-				wasSuccessful := createUser(onPremName, profileName, client, k)
+				//wasSuccessful := createS3User(onPremName, profileName, client, k) // eventually change to createS3User
+				wasSuccessful := true
```

Comment:
I think i did this initially because I didn't want things changing randomly but I suppose we aren't deploying this right away so it would be fine to change it